packages/matrix-client-server/src/rooms/room_information/get_joined_rooms.ts

```diff
@@ -0,0 +1,25 @@
+import type MatrixClientServer from '../../index'
+import { send, type expressAppHandler } from '@twake/utils'
+
+export const getJoinedRooms = (
+  clientServer: MatrixClientServer
+): expressAppHandler => {
+  return (req, res) => {
+    clientServer.authenticate(req, res, (data, id) => {
```

Comment:
authenticate returns a sort of token. Prefer to rename "data" into "token" here to be more clear